osu.Game.Tests/Visual/SongSelectV2/SongSelectTestScene.cs

```diff
@@ -41,6 +42,9 @@ public abstract partial class SongSelectTestScene : ScreenTestScene
         [Cached]
         private readonly ScreenFooter screenFooter;
 
+        [Cached]
+        private readonly ManageCollectionsDialog manageCollectionsDialog;
```

Comment:
Can we not make these things optional so these aren't required everywhere? That's what we usually do.

Reply:
I actually wanted to be able to open the dialog in this test, but I can remove it.

Reply:
Removed in force-pushed changes.